README.md

```diff
@@ -139,7 +123,7 @@ The basic form of feature management is checking if a feature is enabled and the
 …
 IFeatureManager featureManager;
 …
-if (await featureManager.IsEnabledAsync(nameof(MyFeatureFlags.FeatureU)))
+if (await featureManager.IsEnabledAsync(MyFeatureFlags.FeatureU))

```

Comment:
The enum `MyFeatureFlags` is removed, so it should be just string `"FeatureU"`.

Reply:
It's meant to be a static class like [this](https://github.com/jimmyca15/FeatureManagement-Dotnet/blob/user/jimmyca/mReadme3/examples/FeatureFlagDemo/MyFeatureFlags.cs). Do you think it's unintuitive?

Reply:
I see. Then we should have `MyFeatureFlags` defined somewhere in this doc. I would just go with the string directly. It's easier to understand (and less distracting).

Reply:
The only concern I had is that just using IsEnabledAsync(FeatureV) kind of looks like we meant to use a string but forgot quotes.

Reply:
I meant to use the string directly, so It should be `IsEnabledAsync("FeatureV")`.

Reply:
Ah yes, you did put a literal. Okay, I'm fine with that.